src/measure_theory/lebesgue_measure.lean

```diff
@@ -259,6 +260,43 @@ instance locally_finite_volume : locally_finite_measure (volume : measure ℝ) :
   mem_nhds_sets is_open_Ioo ⟨sub_lt_self _ zero_lt_one, lt_add_of_pos_right _ zero_lt_one⟩,
   by simp only [real.volume_Ioo, ennreal.of_real_lt_top]⟩⟩
 
+instance sigma_finite_volume : sigma_finite (volume : measure ℝ) := by apply_instance
```

Comment:
Is this instance necessary?

Reply:
Do we want the sigma-finiteness of the volume on `ι → ℝ` (maybe this was intended as such?)

Reply:
I do not understand why Lean fails to find this instance without a specific hint but without this line some of the next lemmas fail to find `∀ i, sigma_finite (volume : measure ℝ)`.

Reply:
Hmm... That is weird. Maybe it's good to add a comment saying that to the code?

bors d+

Reply:
I removed this line, and the error did not appear.